interfaces/policy/helpers.go

```diff
@@ -221,6 +221,36 @@ func checkSlotConnectionConstraints(connc *ConnectCandidate, cstrs []*asserts.Sl
 	return firstErr
 }
 
+func checkSnapTypeSlotInstallationConstraints1(ic *InstallCandidateMinimalCheck, slot *snap.SlotInfo, cstrs *asserts.SlotInstallationConstraints) error {
+	if err := checkSnapType(slot.Snap, cstrs.SlotSnapTypes); err != nil {
+		return err
+	}
+	if err := checkOnClassic(cstrs.OnClassic); err != nil {
+		return err
+	}
+	return nil
+}
+
+func checkMinimalSlotInstallationConstraints(ic *InstallCandidateMinimalCheck, slot *snap.SlotInfo, cstrs []*asserts.SlotInstallationConstraints) (bool, error) {
+	var firstErr error
+	var hasSnapTypeCstrs bool
```

Comment:
The abbreviation `Cstrs` doesn't immediately imply _constraints_ to me, perhaps it's not worth abbreviating?

Reply:
Renamed.